build.gradle

```diff
@@ -10,7 +10,7 @@ description = "CST is the Cognitive Systems Toolkit, a toolkit for the construct
 
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
-version = '0.5.1'
+version = '0.5.2-beta'
```

Comment:
Following the [Semantic Versioning 2.0.0](https://semver.org/) that we have been using, I recommend this PR to bump CST's version to `0.6.0`, as we effectively have a new feature here.